### ISSUE TITLE
SSL and hyper-parameters for VGG-based model

### ISSUE THREAD

**Issue by viyjy** [liyunsheng13/BDL#25]
Yunsheng,

Would you please provide the SSL model and hyper-parameters for the VGG-based models? Thanks.

**Comment by liyunsheng13** [comment]
You can find the parameters I use to train VGG model from my paper.

**Comment by viyjy** [comment]
Got it. How about the SSL model? Thanks.

**Comment by liyunsheng13** [comment]
Which model do you mean?

**Comment by viyjy** [comment]
I mean the pre-trained SSL model for VGG-based network, which is similar as the [SSL model](https://drive.google.com/open?id=1WtM2dLtRwFvCL_t9Gi6mYhTago_Fk-rA) you provided for DeepLab-based network.

**Comment by liyunsheng13** [comment]
I get it. But I have large numbers of models. I have to spend sometime to find the one you need. Are you in a rush?

**Comment by viyjy** [comment]
Right. It would be very helpful if you can find them. I appreciate it.

**Comment by liyunsheng13** [comment]
I know the CVPR deadline is coming. I'm facing the same problem like you.  I would try to find it as soon as possible. But I don't think you should count on me. I apologize for any inconvenience caused.

**Comment by viyjy** [comment]
I see, thanks.
